packages/hardhat/contracts/strategy/CrabMigration.sol

```diff
@@ -154,15 +158,14 @@ contract CrabMigration is Ownable {
         uint256 crabV1Supply = crabV1.totalSupply();
         (, , uint256 totalCollateral, ) = crabV1.getVaultDetails();
         uint256 amountEthToBorrow = totalCollateral.wmul(crabV1Balance.wdiv(crabV1Supply));
-        bytes memory data;
         euler.deferLiquidityCheck(
             address(this),
             abi.encode(
                 FlashloanCallbackData({
                     caller: msg.sender,
                     amountToBorrow: amountEthToBorrow,
                     callSource: uint8(FLASH_SOURCE.BATCH_MIGRATE),
-                    callData: data
+                    callData: abi.encode(BatchMigrate({strategyCap: _strategyCap}))
```

Comment:
why not just set the strategy cap to be the amount transferred in initialize? Do we need to pass this in?

Reply:
My thinking is that we probably want to set the initial cap > amount to migrate so once we migrate the strategy is live, but it could be done in this way with a call to setStrategyCap() afterwards.

Reply:
ok cool